Resources/Prototypes/Entities/Objects/Fun/toys.yml

```diff
@@ -577,6 +577,20 @@
     containers:
       revolver-ammo: !type:Container
 
+- type: entity
+  parent: RevolverCapGun 
+  id: RevolverCapGunFake
+  name: cap gun
+  description: Looks almost like the real thing! Ages 8 and up.
+  components:
+    - type: RevolverAmmoProvider
+      whitelist:
+        tags:
+          - CartridgeCap
+          - SpeedLoaderCap
+          - CartridgeMagnum
```

Comment:
This should probably start with magnum cartridges already loaded so you don't have to also buy a speedloader to make it usable